### DISCUSSION TITLE
What happened to the recursive folder browse in the nightly?

### DISCUSSION THREAD

**Discussion by gerroon** [bpatrik/pigallery2#749] [Q&A] [answered]
HI

I am testing the nightly dockers. I used to be able to recursively browse folders, but that seems to be gone. Is that removed? Is there a similar function in the new version?

**Comment by bpatrik** [comment]
I'm not sure what you mean. There was no change around folder browsing.

-- Sorry for being brief, sent from my phone.

**Reply by gerroon** [reply]
There was a way to show folders recursively. That option seems to be gone, there is now some kind of group view thing. I am talking about 1x version, I did not pay attention to what it was called. But I was able to show all the pictures in folder like thousands of pictures inside subfolders in a flat view.

**Comment by bpatrik** [comment, marked as answer]
There is a settings for that to enable it. It was always off by default.

-- Sorry for being brief, sent from my phone.

**Reply by gerroon** [reply]
I think you are talking about directory flattening. I never disabled it in the previous version. Not sure why it was disabled then?

**Reply by bpatrik** [reply]
I think the place of the settings moved around that triggered a value reset once you moved to the new version

**Comment by gerroon** [comment]
Hi

It looks like the "Directory flattening" setting is not saved properly. I enable it, save it. Next time I open the setting it is again disabled. I am not sure what the issue isue.

This is what I use "bpatrik/pigallery2:edge-alpine"

**Reply by bpatrik** [reply]
opened: https://github.com/bpatrik/pigallery2/issues/759
